src/notes/note.entity.ts

```diff
@@ -89,9 +93,9 @@ export class Note {
     }
     const newNote = new Note();
     newNote.shortid = shortid;
```

Comment:
Shouldn't this need a default, too?

Reply:
I don't follow, it has one, see https://github.com/hedgedoc/hedgedoc/blob/maintenance/ts_strict_mode/src/notes/note.entity.ts#L91-L93

Reply:
my bad